doc/data/messages/i/invalid-enum-extension/bad.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class A(Enum):
+    x = 1
+    y = 2
+
+
+class B(A):  # [invalid-enum-extension]
+    z = 3
```

Comment:
```suggestion
from enum import Enum


class Color(Enum):
    ORANGE = 1
    CHERRY = 2


class Fruit(Color):  # [invalid-enum-extension]
    APPLE = 3
```

Reply:
Haha yeah I like the more interesting naming!